src/main/resources/db/migration/V1__init_db.sql

```diff
@@ -1,159 +1,292 @@
-SET STATEMENT_TIMEOUT = 0;
-SET LOCK_TIMEOUT = 0;
-SET IDLE_IN_TRANSACTION_SESSION_TIMEOUT = 0;
-SET CLIENT_ENCODING = 'UTF8';
-SET STANDARD_CONFORMING_STRINGS = ON;
-SELECT PG_CATALOG.SET_CONFIG('SEARCH_PATH', '', FALSE);
-SET CHECK_FUNCTION_BODIES = FALSE;
-SET XMLOPTION = CONTENT;
-SET CLIENT_MIN_MESSAGES = WARNING;
-SET ROW_SECURITY = OFF;
+create table flyway_schema_history
+(
+	installed_rank integer not null
+		constraint flyway_schema_history_pk
+			primary key,
+	version varchar(50),
+	description varchar(200) not null,
+	type varchar(20) not null,
+	script varchar(1000) not null,
+	checksum integer,
+	installed_by varchar(100) not null,
+	installed_on timestamp default now() not null,
+	execution_time integer not null,
+	success boolean not null
+);
 
-SET DEFAULT_TABLESPACE = '';
+alter table flyway_schema_history owner to postgres;
 
-SET DEFAULT_TABLE_ACCESS_METHOD = HEAP;
+create index flyway_schema_history_s_idx
+	on flyway_schema_history (success);
 
-CREATE TABLE PUBLIC.USERS (
-  USER_ID SERIAL PRIMARY KEY,
-  EMAIL VARCHAR,
-  PASSWORD VARCHAR,
-  STATUS VARCHAR,
-  FIRST_NAME VARCHAR,
-  LAST_NAME VARCHAR,
-  QUALIFICATION VARCHAR,
-  PHONE VARCHAR,
-  BIO TEXT,
-  CREATED_AT TIMESTAMP DEFAULT (NOW())
+create table roles
+(
+	role_id serial not null
+		constraint roles_pkey
+			primary key,
+	role_name varchar
 );
 
-CREATE TABLE PUBLIC.ROLES (
-  ROLE_ID SERIAL PRIMARY KEY,
-  ROLE_NAME VARCHAR
+alter table roles owner to postgres;
+
+create table users
+(
+	user_id serial not null
+		constraint users_pkey
+			primary key,
+	email varchar,
+	password varchar,
+	status varchar,
+	first_name varchar,
+	last_name varchar,
+	phone varchar,
+	created_at timestamp default now(),
+	avatar varchar,
+	enabled boolean,
+	role_id integer
+		constraint users_role_id_fkey
+			references roles
 );
 
-CREATE TABLE PUBLIC.ROLES_USERS (
-  ROLE_ID INT,
-  USER_ID INT
-);
+alter table users owner to postgres;
 
-CREATE TABLE PUBLIC.REGIONS (
-  REGION_ID SERIAL PRIMARY KEY,
-  NAME VARCHAR
+create table regions
+(
+	region_id serial not null
+		constraint regions_pkey
+			primary key,
+	name varchar
 );
 
-CREATE TABLE PUBLIC.INSTITUTIONS (
-  INSTITUTION_ID SERIAL PRIMARY KEY,
-  NAME VARCHAR,
-  REGION_ID INT,
-  ADDRESS VARCHAR
-);
+alter table regions owner to postgres;
 
-CREATE TABLE PUBLIC.USERS_INSTITUTIONS (
-  USER_ID INT,
-  INSTITUTION_ID INT,
-  IS_PRIMARY BOOLEAN
+create table tags
+(
+	tag_id serial not null
+		constraint tags_pkey
+			primary key,
+	tag varchar
+		constraint unique_name
+			unique
 );
```

Comment:
Maybe would be better If you apply code format for all scripts? Does it make sense?